doc/pages/user/080-grafana_datasource.md

```diff
@@ -1 +1,49 @@
-# Using Astarte's Grafana Datasource
+# Astarte Datasource Plugin for Grafana
+
+Astarte Datasource Plugin conveys data from Astarte to
+[Grafana](https://github.com/grafana/grafana), the open-source platform for
+monitoring and observability, developed by [Grafana Labs](https://grafana.com/).
+Actual data visualisation is provided by Grafana via its
+[plugins](https://grafana.com/grafana/plugins/?type=panel).
+
+You can browse the source code of this plugin on its
+[GitHub repository](https://github.com/astarte-platform/grafana-astarte-datasource).
+
+![Sample Grafana dashboard](assets/astarte-grafana-dashboard-overview.png)
+
+## Try it!
+
+When deploying locally using `docker-compose` as mentioned in the
+[Astarte in 5 mins tutorial](https://docs.astarte-platform.org/1.0/010-astarte_in_5_minutes.html#install-astarte),
+Astarte Datasource Plugin will be automatically installed. You may then access Grafana
+by visiting http://localhost:3000.
+
+When first logging into Grafana, you will be prompted to change default
+credentials user `admin`, password: `admin`.
+
+# Configure the datasource
+
+In order to get data from Astarte, you will need to create a new datasource
+referring to your own Astarte instance.
+
+![Add a Grafana Datasource](assets/astarte-grafana-add-datasource.png)
+
+You will need to provide Astarte API URLs, the realm name and Astarte AppEngine token.
+
+![Datasource configuration fields](assets/astarte-grafana-datasource-fields.png)
```

Comment:
Could it be useful to carry on the example drawn from 'Astarte in 5 minutes' and directly display sample endpoint/config that works with the default local Astarte instance? Instead of showing random config

Reply:
the plugin doesn't support "localhost" as a valid url for now. We may add one when it's supported